src/providers/ethereum.ts

```diff
@@ -186,6 +188,11 @@ export default class EthereumProvider extends BloctoProvider implements Ethereum
           result = await this.handleSign(payload);
           break;
         }
+        case 'wallet_disconnect': {
+          removeItem(KEY_SESSION);
+          result = null;
```

Comment:
should also clear code/accounts etc in the class

Reply:
done